vars/folioHelm.groovy

```diff
@@ -198,9 +198,8 @@ String generateModuleValues(RancherNamespace ns, String moduleName, String modul
         moduleConfig <<
           [
             [eureka: [enabled         : true,
-                      sidecarContainer: [image: "${Constants.ECR_FOLIO_REPOSITORY}/folio-module-sidecar",
-                                         tag  : ns.getModules().allModules['folio-module-sidecar']]]]
-
+                      sidecarContainer: [ image: "${repository}/folio-module-sidecar",
```

Comment:
Please consider adding a separate repository name determination for the sidecar since it's a standalone image with its own version.

Reply:
@epam-avramenko
Do you want to say we can have 'mod-users-keycloak' image from 'folioorg' repository
and its 'folio-module-sidecar' image from 'folioci' repository in the same time?

Reply:
Yep. Because folio-module-sidecar is a separate image from a separate repo utilized by all modules, it has a development process separated from them.

Reply:
Fixed